### DISCUSSION TITLE
Can't build latest release

### DISCUSSION THREAD

**Discussion by bensuffolk** [HASwitchPlate/openHASP#339] [Q&A] [answered]
I cloned the repository a week or so ago and was able to build (the lanbon_l8) binaries just fine, but I forked the repository so I could start to make a couple of changes, and I find that my new fork can't build.

So I have gone back and cloned a complete new on again and that won't build either. This is the error 👍 

```src/hasp/hasp_font.cpp:11:10: fatal error: ft2build.h: No such file or directory```

It seems I have lib/freetype in the one from a week or so ago, buts missing in the new ones.

**Comment by nagyrobi** [comment, marked as answer]
From the [docs](https://www.openhasp.com/0.6.3/compiling/local/):
also run `git submodule update --init --recursive`

**Reply by bensuffolk** [reply]
Ah yes, I remember doing that last time round. Now I feel silly :)
